### PR TITLE
Pd 160 update pxctl cluster optimized restore

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: content/reference/CLI/cluster.md [hunk 1]
```diff
@@ -167,3 +167,117 @@ USAGE:
 OPTIONS:
    --io_priority value  IO Priority: [high|medium|low] (default: "low")
 ```
+
+## Enabling optimized restores
+
```
Comment (talakad) [inline content/reference/CLI/cluster.md]: May be add link here that explains optimized restores. i think that is in cloud-snap restore reference.

Comment (andreipope) [reply content/reference/CLI/cluster.md]: @talakad Updated. My understanding is that there are two ways in which you can enable optimized restores:

1. set the runtime option `optimized_restore=1` while starting the Portworx container

2. add the setting to the cluster spec and then use the `pxctl cluster options` command to enable or disable optimized restores.

Let me know if this needs to be changed.

Comment (talakad) [reply content/reference/CLI/cluster.md]: Thanks Andrei!. Lets remove the runtime option for optimized totally form the documentation pages as that is not clean way for users to enable the optimized restores.

File: content/reference/CLI/shared/optimized-restores-definition.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+keywords: portworx, pxctl, restore
+description: Explains what is an optimized restore
+---
+
+With PX-Enterprise 2.1.0, users can choose to do optimized restores.  The way this works is that every successful restore creates a snapshot that will be used for the next incremental restore of the same volume.  Hence, for an incremental restore, only the last incremental backup will be downloaded instead of downloading all the dependent backups. Optimized restores are especially useful for workflows that involve frequent restores from a different cluster.
+However, this works only if dependent backups were downloaded previously.
+
+Currently, to enable this functionality, you have to:
+
+- set the runtime option `optimized_restore=1` while starting the Portworx container
```
Comment (talakad) [inline content/reference/CLI/shared/optimized-restores-definition.md]: Remove this line. Let force users to use this through Pxctl.

**Pull request merged**